lab-scott/controllers/pic-controller.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const Pic = require('../models/pic')
+const Gallery = require('../models/gallery')
+const mongoose = require('mongoose')
+const dataDir = `${__dirname}/../data`
+const multer = require('multer')
+const upload = require({dest: dataDir})
+const del = require('del')
+const AWS = require('aws-sdk')
+const path = require('path')
+const fs = require('fs')
+const createError = require('http-errors')
+
+module.exports = exports = {}
+
```

Comment:
AWS.config.setPromisesDependency(require('bluebird'));
const s3 = new AWS.S3();   

added this and your test files ran